ENV.md

```
@@ -0,0 +1,136 @@
+|------------|-------------|-------|
+| `EXPERIMENTAL_CACHE` | `CACHE_ENABLED` | |
+| `EXPERIMENTAL_CACHE_L1_MAX_SIZE` | `CACHE_L1_MAX_SIZE` | |
+| `EXPERIMENTAL_CACHE_L2_REDIS_URL` | `CACHE_L2_REDIS_URL` | |
+| `EXPERIMENTAL_CACHE_REDIS_BUS_HOST` | `CACHE_BUS_HOST` | |
+| `EXPERIMENTAL_CACHE_REDIS_BUS_PORT` | `CACHE_BUS_PORT` | |
+| `EXPERIMENTAL_CACHE_DEBUG` | `CACHE_DEBUG` | |
+| `OAUTH_GOOGLE_SECRET_ID` | `OAUTH_GOOGLE_CLIENT_SECRET` | Renamed for consistency |
+| `OAUTH_GITHUB_SECRET_ID` | `OAUTH_GITHUB_CLIENT_SECRET` | Renamed for consistency |
+
+Using deprecated variables will log a warning:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)